src/auth/auth-service.ts

```diff
@@ -15,15 +16,16 @@ export function initAuth() {
 
   const { token, refreshToken } = loadTokens();
 
-  return kc.init({
+  await kc.init({
```

Comment:
Init should resolve to a boolean property. Perhaps it's easier to "return kc.init(..." instead of awaiting init and then checking authenticated status?

Reply:
hmm, jeg tenkte vi l;ste den.. skal sjekke

Reply:
kc.authenticated er boolean